jiant/models.py

```diff
@@ -158,8 +158,10 @@ def build_sent_encoder(args, vocab, d_emb, tasks, embedder, cove_layer):
         )
         d_sent = args.d_word
         log.info("Using PRPN sentence encoder!")
-    elif any(isinstance(task, LanguageModelingTask) for task in tasks) or args.sent_enc == "bilm":
-        assert_for_log(args.sent_enc in ["rnn", "bilm"], "Only RNNLM supported!")
+    elif any(isinstance(task, LanguageModelingTask) for task in tasks):
```

Comment:
wait, how do we deal with language modeling task when we have a input_module like bert? do we have that supported now?

Reply:
Yes that is supported